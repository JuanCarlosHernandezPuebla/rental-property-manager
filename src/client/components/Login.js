import React, { Component } from 'react';
import { Link } from 'react-router-dom';

class Login extends Component {

    render() {
        return(
            <div className="container">
                <form className="form-login" method="POST">
                    <img className="mb-4" src="/images/rpm_logo.png" alt="" width="72" height="72" />
                    <h1 className="h3 mb-3 font-weight-normal">Please log in</h1>
                    <label htmlFor="inputUsername" className="sr-only">Username</label>
                    <input type="text" id="inputUsername" name="username" className="form-control" placeholder="Username" required autoFocus />
                    <label htmlFor="inputPassword" className="sr-only">Password</label>
                    <input type="password" id="inputPassword" name="password" className="form-control" placeholder="Password" required />
                    <button className="btn btn-lg btn-primary btn-block btn-login" type="submit">Log in</button>
                    <div>
                        <Link className="nav-link" to="/create">Create an account</Link>
                    </div>
                </form>
            </div>     
        );
    }

}

export default Login;