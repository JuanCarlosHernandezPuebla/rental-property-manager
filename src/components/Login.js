import React, { Component } from 'react';
import LoginForm from './Forms/LoginForm';
import { Link } from 'react-router-dom';

export default class Login extends Component {

  render() {
    return (
      <div className="container">
        <div className="login-form-content">
          <LoginForm />
          <div className="text-center">
            <Link className="nav-link" to="/create">Create an account</Link>
          </div>
        </div>
      </div>
    );
  }

}
