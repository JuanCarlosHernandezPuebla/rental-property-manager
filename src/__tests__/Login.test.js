import React from 'react';
import { shallow } from 'enzyme';
import Login from '../components/Login';
import renderToJson from 'enzyme-to-json';

test('Login render test', () => {
    const component = shallow(<Login/>);
    expect(renderToJson(component)).toMatchSnapshot();
});