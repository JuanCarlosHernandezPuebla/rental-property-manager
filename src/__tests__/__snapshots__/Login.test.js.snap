// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login render test 1`] = `
<div
  className="container"
>
  <LoginForm />
</div>
`;
